### PR TITLE
Added Global Exception Handler for Web API

### PR DESCRIPTION
**Pull request by Mazar-Shaik**
…andler Interface

### CODE REVIEW

File: src/EPR.Calculator.API/Program.cs [hunk 1]
```diff
@@ -32,6 +33,7 @@
     app.UseSwaggerUI();
 }
 
+app.UseExceptionHandler( _ => { });
```
Comment (udaydefra) [inline src/EPR.Calculator.API/Program.cs:36]: Can you please add a comment and also reply the reference to the URL

Comment (Mazar-Shaik) [reply src/EPR.Calculator.API/Program.cs:36]: If we wont add this lambda expression then it throws this exception and gives the below answer to solve.
An error occurred when configuring the exception handler middleware. Either the '_ExceptionHandlingPath_' or the '_ExceptionHandler_' property must be set in 'UseExceptionHandler()'. Alternatively, set one of the aforementioned properties in 'Startup.ConfigureServices' as follows: '**services.AddExceptionHandler(options => { ... });**' or configure to generate a 'ProblemDetails' response in 'service.AddProblemDetails()'.

We can use any one of these to fix.
services.AddExceptionHandler(options => { ... }); -> I used this.

or
combination of these 2 lines
service.AddProblemDetails();
services.AddExceptionHandler();

or app.UseExceptionHandler("/Error") for _ExceptionHandlingPath_ but we don't have any redirect to an error-handling path in web api project.

It’s good to specify either a custom error path ( we don't have now ) or inline error handling logic ( but we we are doing this logic in separate file which is global exception handling file)

Comment (udaydefra) [reply src/EPR.Calculator.API/Program.cs:36]: Please give a URL reference @Mazar-Shaik

Comment (Mazar-Shaik) [reply src/EPR.Calculator.API/Program.cs:36]: https://learn.microsoft.com/en-us/aspnet/core/web-api/handle-errors?view=aspnetcore-8.0

Comment (udaydefra) [reply src/EPR.Calculator.API/Program.cs:36]: I can see usage for 
`app.UseExceptionHandler();`
If you agree please change it and remove the blank arrow function @Mazar-Shaik

Comment (Mazar-Shaik) [reply src/EPR.Calculator.API/Program.cs:36]: @udaydefra Okay sure I will change but we need to add builder.Services.AddProblemDetails(); as well to work

So 2 lines I'm adding now.
builder.Services.AddProblemDetails()
app.UseExceptionHandler();

**Review by udaydefra — APPROVED** [review]
@Mazar-Shaik  Very good work! Thanks

**Pull request merged**